tests/raftstore/cluster.rs

```diff
@@ -142,21 +142,24 @@ impl<T: Simulator> Cluster<T> {
                         request: RaftCmdRequest,
                         timeout: Duration)
                         -> Result<RaftCmdResponse> {
-        let t = Instant::now();
         self.sim
             .rl()
             .call_command(request, timeout)
-            .map_err(|e| box_err!("call command failed {:?}, cost {:?}", e, t.elapsed()))
     }
 
     pub fn call_command_on_leader(&mut self,
                                   region_id: u64,
                                   mut request: RaftCmdRequest,
                                   timeout: Duration)
                                   -> Result<RaftCmdResponse> {
-        let leader = self.leader_of_region(region_id).unwrap();
-        request.mut_header().set_peer(leader);
-        self.call_command(request, timeout)
+        for _ in 0..200 {
```

Comment:
I agree. 
but we have already tons of code call_command_on_leader().unwrap() like patten, make CI vulnerable. we should clean up code in following PRs...